lib/runner.ex

```diff
@@ -30,14 +42,16 @@ defmodule Runner do
     modules
     |> Enum.drop_while( &(&1 != start_module))
     |> Enum.take_while( &(run_module(&1) == :passed))
+    |> done?
   end
 
+  defp done?(modules), do: modules == @modules
```

Comment:
`done?` Needs to be cleverer than that. If we start from somewhere in the middle of the koans (using `--koans=<Koan>`)  then `#drop_while(&(&1 != start_module))` will remove a bunch of modules.
That means `done?(module)` will never be `true`.

I am trying to come up with checks that will tell us when the user is done, but I am having a hard time...